prow/cmd/crier/README.md

```diff
@@ -48,10 +48,49 @@ The actual report logic is in the [github report library](/prow/github/report) f
 
 You can enable the Slack reporter in crier by specifying the `--slack-workers=n` and `--slack-token-file=path-to-tokenfile` flags.
 
-The `--slack-token-file` flag takes a path to a file containing a Slack [*OAuth Access Token*](https://api.slack.com/docs/oauth). The access token can be obtained after installing 
-a [Slack app](https://api.slack.com/apps) for the *workspace*. The app should have the [`chat:write:bot`](https://api.slack.com/scopes/chat:write:bot) scope granted in order to [post messages](https://api.slack.com/methods/chat.postMessage) to the configured Slack *channel*.
+The `--slack-token-file` flag takes a path to a file containing a Slack [**OAuth Access Token**](https://api.slack.com/docs/oauth).
+
+The **OAuth Access Token** can be obtained as follows:
+1. Navigate to: https://api.slack.com/apps.
+1. Click **Create New App**; 
+1. Provide an **App Name** (e.g. Prow Slack Reporter) and **Development Slack Workspace** (e.g. Kubernetes).
+1. Click **Permissions**.
+1. Add the `chat:write:bot` scope using the **Scopes** dropdown and **Save Changes**.
+1. Click **Install App to Workspace**
+1. Click **Allow** to authorize the Oauth scopes.
+1. Copy the **OAuth Access Token** and save the token to a file.
+
+Once the *access token* is obtained and saved to a file, you can create a `secret` in the cluster using that file: 
+
+```shell
+kubectl create secret generic slack-token --from-literal=token=/path/to/slack/token/file
```

Comment:
Now it has to be `--from-file` :grimacing:

Reply:
🤦‍♂. Thanks.